src/config/dynamic-theme-fixes.config

```diff
@@ -12488,6 +12488,13 @@ img[alt="Jewish Currents"]
 
 ================================
 
+jigsaw.yuzu.com
```

Comment:
Can't check what exactly is fixed without account. Can you share at least screens before and after fix?